File: src/com/mrpowergamerbr/loritta/utils/music/AudioTrackWrapper.java
package com.mrpowergamerbr.loritta.utils.music;

import com.sedmelluq.discord.lavaplayer.track.AudioTrack;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
public class AudioTrackWrapper {
	private AudioTrack track;
	private boolean isAutoPlay;
}

File: src/main/java/com/mrpowergamerbr/loritta/audio/AudioTrackWrapper.kt
package com.mrpowergamerbr.loritta.audio

import com.sedmelluq.discord.lavaplayer.track.AudioTrack
import net.dv8tion.jda.api.entities.User

data class AudioTrackWrapper(
		val track: AudioTrack,
		val isAutoPlay: Boolean = false,
		val user: User,
		val metadata: MutableMap<String, String> = mutableMapOf())